app/uk/gov/hmrc/filetransmission/connector/MdgConnector.scala

```diff
@@ -72,12 +72,12 @@ class MdgConnector @Inject()(
         case s if Status.isClientError(s) =>
           logger.warn(
             s"Sending request for file with reference [${request.file.reference}] failed. MDG Correlation id [$correlationId]. Cause [${response.body}]")
-          MdgRequestFatalError(response.body)
+          MdgRequestFatalError(HttpErrorFunctions.badRequestMessage("POST", serviceConfiguration.mdgEndpoint, response.body))
```

Comment:
This message will say it was a 400, when it wasn't necessarily?
Is it coupling to http-verbs unnecessarily?